ui/src/components/SampleGrid/ClearQueueDialog.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ConfirmActionDialog from '../GenericDialog/ConfirmActionDialog';
+import { useDispatch, useSelector } from 'react-redux';
+import { clearQueue } from '../../actions/queue';
+import { showConfirmClearQueueDialog } from '../../actions/general';
+
+function ClearQueueDialog() {
+  const dispatch = useDispatch();
+  const show = useSelector(
+    (state) => state.general.showConfirmClearQueueDialog,
+  );
+
+  return (
+    <ConfirmActionDialog
+      title="Clear sample list?"
+      okBtnLabel="Clear"
+      show={show}
+      onOk={() => dispatch(clearQueue())}
+      onHide={() => dispatch(showConfirmClearQueueDialog(false))}
+    >
+      This will also <strong>clear the queue</strong> and{' '}
+      <strong>unmount</strong> the current sample.
```

Comment:
do we actually umount the sample? or it simply dissapears from the UI?

Reply:
Excellent question :smile:

Reply:
Right, it doesn't. If you sync again, the sample reappears.

Reply:
I've reworded. I think clearing the mounted sample from the UI is less than ideal but this can be discussed in #1420.